extras/kubernetes/heketi.json

```diff
@@ -0,0 +1,44 @@
+{
+  "_port_comment": "Heketi Server Port Number",
+  "port": "8080",
+
+  "_use_auth": "Enable JWT authorization. Please enable for deployment",
+  "use_auth": false,
+
+  "_jwt": "Private keys for access",
+  "jwt": {
+    "_admin": "Admin has access to all APIs",
+    "admin": {
+      "key": "My Secret"
+    },
+    "_user": "User only has access to /volumes endpoint",
+    "user": {
+      "key": "My Secret"
+    }
+  },
+
+  "_glusterfs_comment": "GlusterFS Configuration",
+  "glusterfs": {
+    "_executor_comment": "Execute plugin. Possible choices: mock, kubernetes, ssh",
+    "executor": "kubernetes",
+
+    "_db_comment": "Database file name",
+    "db": "/var/lib/heketi/heketi.db",
+
+    "kubeexec": {
+      "rebalance_on_expansion": true
+    },
+
+    "sshexec": {
+      "rebalance_on_expansion": true,
+      "keyfile": "/etc/heketi/private_key",
+      "fstab": "/etc/fstab",
+      "port": "22",
+      "user": "root",
+      "sudo": false
+    }
+  },
+
+  "_backup_db_to_kube_secret": "Backup the heketi database to a Kubernetes secret when running in Kubernetes. Default is off.",
```

Comment:
There are two more config options that come here. Refer to https://github.com/gluster/gluster-kubernetes/pull/285/files#diff-9a4b889b4e6db0956590de1bc7f5d067R41 for those block related options.

The PR in gluster-kubernetes isn't merged yet but heketi has that option already. I leave it to you if you want to add those later after the PR is merged.

Reply:
I'd prefer to do it afterward. It's probably good to have a more complete configuration but since the system works without it, I would defer that improvement for now.

Reply:
Technically, the options are not from the gluster-kubernetes repo, but come from heketi itself: They are also in the file `etc/heketi.json` in this repo. But the requiredl container updates for gluster (adding gluster-blockd ....) are still pending, hence the gluster-kubernetes pr is not merged yet. We could still add the config options to the heketi.json for this example now. But as you say, we can as well add it later...

Reply:
The only thing I'm kind of uncertain about is: do we want to add any of those artifacts (config, templates etc) here at all?... But we can also still remove stuff later if we want. :-)